easybuild/easyconfigs/c/CPC2/CPC2-0.1-foss-2022a.eb

```
@@ -0,0 +1,40 @@
+# Author: Samuel Moors, Vrije Universiteit Brussel (VUB)
+# Updated: Denis Kristak (Inuits)
+# Update to a Python3 version
+easyblock = 'PackedBinary'
+
+name = 'CPC2'
+version = '0.1'
+local_commit = '813487e'
+
+homepage = 'http://cpc2.cbi.pku.edu.cn'
+description = """ a fast and accurate coding potential calculator based on sequence intrinsic features  """
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+# original sources at http://cpc2.cbi.pku.edu.cn/download.php are currently not available
+# download from github mirror
+source_urls = ['https://github.com/lucventurini/CPC2/archive/']
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+patches = ['%(name)s-%(version)s_fix_libsvm_path_and_python2.patch']
+checksums = [
+    {'CPC2-0.1.tar.gz': '6fd4f3fb05b179073709c033481ae082085eabc7c5281cd743c9419d3ec6de67'},
+    {'CPC2-0.1_fix_libsvm_path_and_python2.patch': '657f552ebfad925e8c051a19426cd5bb1b8ad7da8cdcc3e701e1220673330842'},
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('LIBSVM', '3.30'),
+    ('Biopython', '1.79'),
+]
+
+install_cmd = 'cp -a CPC2*/{bin,data} %(installdir)s/'
+
+sanity_check_paths = {
+    'files': ['bin/CPC2.py'],
+    'dirs': [],
```

also check for `data` dir?

```suggestion
    'dirs': ['data'],